build_library/grub.cfg

```diff
@@ -47,28 +50,34 @@ function load_coreos {
         set suf=""
     fi
 
-    echo "Loading ($root)/boot/vmlinuz"
-    linux$suf /boot/vmlinuz $linux_console $linux_root "$@" $linux_append
+    search --no-floppy --set root --part-label EFI-SYSTEM
```

Comment:
$root is initially set th the ESP in load.cfg:

```
search.fs_uuid ${ESP_FSID} root \$root
```

the gptprio.next calls should be updated to set $usr or similar instead of $root